.travis.yml

```diff
@@ -11,11 +11,6 @@ cache:
 services:
   - docker
 
-# whitelist
```

Comment:
@gerring In the .travis.yml you added a whitelist for branches. I would like to remove that restriction as I want all branches to build without fiddling, including branches in my repo before I have created PR. Any objections to removing it? I was wondering if you had a bad experience without it that I don't want to repeat here.